axum-handle-error-extract/src/lib.rs

```diff
@@ -125,7 +125,6 @@
     clippy::option_option,
     clippy::verbose_file_reads,
     clippy::unnested_or_patterns,
-    clippy::self_named_module_files,
```

Comment:
Why did you remove this?

Reply:
I got

```
warning: unknown lint: `clippy::self_named_module_files`
   --> axum-handle-error-extract/src/lib.rs:128:5
    |
128 |     clippy::self_named_module_files,
    |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ help: did you mean: `clippy::self_named_constructors`
    |
    = note: `#[warn(unknown_lints)]` on by default
```

So I guess its a nightly lint and not sure we should have nightly lints disabled for contributors working on stable.

Reply:
Wdym. by having lints disabled for contributors? People could "just" ignore the warning when working on stable.

Reply:
I still get the warning when I'm hacking on the code and I pull the "show me all the warnings window". I assume getting rid of the warning would require running nightly.

Reply:
I don't think telling people to "just ignore the warnings" is great 😕

Reply:
> I assume getting rid of the warning would require running nightly.

You could also `#[allow(unknown_lints)]` but that obviously has its downsides as well.

Anyways, I can understand not wanting to enable lints that stable clippy doesn't recognize.